sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBase.java

```diff
@@ -563,7 +563,7 @@ AccessToken getTokenFromAzureDeveloperCLIAuthentication(StringBuilder azdCommand
                                                 +
                                                 "To mitigate this issue, please refer to the troubleshooting guidelines here at "
                                                 +
-                                                "https://aka.ms/azsdk/java/identity/azclicredential/troubleshoot"));
```

Comment:
Did you accidentally have the azcli url before?

Reply:
yeah, it was copy pasted in the original PR before